mappers/bluetooth_mapper/action_manager/action_manager.go

```diff
@@ -21,7 +21,8 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/golang/glog"
+	"k8s.io/klog"
+
```

Comment:
Do we need a blank line here ? paypal/gatt is also a vendor library, so i think both of them can be part of the same sub-section

Reply:
done